app/lib/account/auth_provider.dart

```diff
@@ -8,11 +8,17 @@ class AuthResult {
   final String oauthUserId;
   final String email;
   final String audience;
+  final String? nonce;
```

Comment:
Do we need nonce here? After OIDC flow have checked it, I don't think we need to check it again.